superset/assets/src/dashboard/util/getPublishSubscriberMap.js

```diff
@@ -119,10 +119,26 @@ function getSubscriberMap(slices, publishers) {
 }
 
 export default function getPublishSubscriberMap(slices) {
+    slices.forEach(slice => {
+        let linkedSlices = getLinkedSlicesFromSubscriberLayer(slice.formData.subscriber_layers);
+        slice.formData.linked_slice = linkedSlices ? linkedSlices : [];
+        slice.formData.actions = ['APPLY_FILTER'];
```

Comment:
it will not support backward compatibility  as we are not retain old ones in case no subscription layer

Reply:
this will be implemented when we refactor

Reply:
done